backbone/src/main/java/org/researchstack/backbone/ui/ConsentTaskActivity.kt

```diff
@@ -96,9 +97,7 @@ class ConsentTaskActivity : TaskActivity() {
 
         val consentAssetsFolder = intent.getStringExtra(EXTRA_ASSETS_FOLDER)
         val role = LocalizationUtils.getLocalizedString(this, R.string.rsb_consent_role)
-        val df = DateFormat.getDateInstance(DateFormat.MEDIUM,
-                LocalizationUtils.getLocaleFromString(
-                        LocalizationUtils.getPreferredLocale(this)))
+        val df = SimpleDateFormat("yyyy-MM-dd HH:mm:ss Z", Locale.getDefault())
```

Comment:
why `Locale.getDefault()` instead of getting the user's current locale?

Reply:
Do we want to add another instance of SimpleDateFormat after all we've been discussing? is it really needed?

Reply:
I fixed it and the timezone too